txp/Pos/Script/Examples.hs

```diff
@@ -80,6 +81,16 @@ intValidator = fromE $ parseValidator [text|
             False -> failure } }
     |]
 
+intValidatorWithVersionOne :: Script
+intValidatorWithVersionOne = fromE $ parseValidatorWithVersionOne [text|
+    data Foo = { Foo }
+
+    validator : Int -> Comp Foo {
+        validator x = case !equalsInt x 1 of {
+            True  -> success Foo ;
+            False -> failure } }
+    |]
```

Comment:
It's copy-pasted, please avoid this copy-paste. You can make `setVersionOne` a standalone exported function and use it here.